javascript/house-style.md

```diff
@@ -692,20 +692,47 @@ export {init};
 
 #### Events for unrelated modules
 
-Use a small Publish Subscribe implementation, like [PubSubJS](https://github.com/mroderick/PubSubJS):
+Use the [`createEvent` module](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-javascript#createevent) from the Springer Nature Elements toolkit. This module functions as a namespaced wrapper around Javascript `customEvent`.
 
-```js
-// analytics.js
+One limitation of using the existing infrastructure that Javascript provides — the ability to create an `Event` and consume using `addEventListener` — instead of a [PubSub](https://addyosmani.com/blog/understanding-the-publishsubscribe-pattern-for-greater-javascript-scalability/) style implementation, is that you have to hang the event handler off a `DOM Element` or the `window`.
 
-import {PubSub} from 'pub-sub';
+In the case of two unrelated modules, this would mean that if `moduleA` publishes an event, then `moduleB` would have no knowledge of the DOM element that the event is bound to. This is solved by having your application manage event listeners, and keeping your modules fully isolated with no knowledge of other modules; for example:
 
-function beaconToAnalytics(event) {}
+```js
+// Module A
+// Relevant excerpt only
+const event = createEvent('event', 'component', {
+	detail: {
+        hazcheeseburger: true
+    }
```

Comment:
```suggestion
    detail: {
        hazcheeseburger: true
    }
```

Also the code samples in this document in general use a mixture of tabs, 2-char soft tabs and 4-char soft tabs.  Was that intentional, with the concerns in the samples being independent of indentation concerns, or is it an oversight?

Reply:
Hey @danyalaytekin, sorry been on holiday. Thanks for the review 😄  This mixture of spacing was not intentional. I will go through and standardise.